board.go

```diff
@@ -149,9 +157,10 @@ func (s *BoardService) DeleteBoard(boardID int) (*Board, *Response, error) {
 // This only includes sprints that the user has permission to view.
 //
 // JIRA API docs: https://docs.atlassian.com/jira-software/REST/cloud/#agile/1.0/board/{boardId}/sprint
-func (s *BoardService) GetAllSprints(boardID string) ([]Sprint, *Response, error) {
-	apiEndpoint := fmt.Sprintf("rest/agile/1.0/board/%s/sprint", boardID)
-	req, err := s.client.NewRequest("GET", apiEndpoint, nil)
+func (s *BoardService) GetAllSprints(boardID int, options *GetAllSprintsOptions) ([]Sprint, *Response, error) {
```

Comment:
Can you create a different method that is `GetAllSprintsWithOptions` so it doesn't break the interface for older users?

Reply:
yeah I can do that

Reply:
I put up a new set of changes. take a look